core/platform/email/gae_email_services.py

```diff
@@ -34,23 +43,22 @@ def send_mail(
     email_manager._send_email().
 
     Args:
-      - sender_email: str. the email address of the sender. This should be in
-          the form 'SENDER_NAME <SENDER_EMAIL_ADDRESS>'.
-      - recipient_email: str. the email address of the recipient.
-      - subject: str. The subject line of the email.
-      - plaintext_body: str. The plaintext body of the email.
-      - html_body: str. The HTML body of the email. Must fit in a datastore
-          entity.
-      - bcc_admin: bool. Whether to bcc feconf.ADMIN_EMAIL_ADDRESS on the email.
-      - reply_to_id: str or None. The unique reply-to id used in reply-to email
-          sent to recipient.
+        sender_email: str. The email address of the sender. This should be in
+            the form 'SENDER_NAME <SENDER_EMAIL_ADDRESS>'.
+        recipient_email: str. The email address of the recipient.
+        subject: str. The subject line of the email.
+        plaintext_body: str. The plaintext body of the email.
+        html_body: str. The HTML body of the email. Must fit in a datastore
+            entity.
+        bcc_admin: bool. Whether to bcc feconf.ADMIN_EMAIL_ADDRESS on the email.
+        reply_to_id: str or None. The unique reply-to id used in reply-to email
+            sent to recipient.
 
     Raises:
-      Exception: if the configuration in feconf.py forbids emails from being
-        sent.
-      ValueError: if 'sender_email' or 'recipient_email' is invalid, according
-        to App Engine.
-      (and possibly other exceptions, due to mail.send_mail() failures).
+        Exception: If the configuration in feconf.py forbids emails from being
+            sent.
+        ValueError: If 'sender_email' or 'recipient_email' is invalid, according
```

Comment:
why have you changed the order again? (asked [here](https://github.com/oppia/oppia/pull/4795#discussion_r174169309))
It was correct in earlier commits, but I cannot see those commits (only one commit in the list). Are you using `amend`?

Reply:
Yes, I am amending the changes.